src/components/tiles/table/cell-text-editor.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useEffect, useRef, useState, KeyboardEvent, useContext } from "react";
 import { EditorProps } from "react-data-grid";
-import { Portal } from "@chakra-ui/react";
 import TextareaAutosize from "react-textarea-autosize";
+import { createPortal } from "react-dom";
```

Comment:
I couldn't get this to work with Chakra UI's `Portal`. It seems like that should be possible, though. I may try again if this solution seems promising.